clusterloader2/pkg/measurement/common/ooms_tracker.go

```diff
@@ -124,13 +182,16 @@ func (m *clusterOOMsTrackerMeasurement) start(config *measurement.Config) error
 	if err := m.initFields(config); err != nil {
 		return fmt.Errorf("problem with OOMs tracking measurement fields initialization: %w", err)
 	}
+	ctx := context.Background()
+	client := config.ClusterFramework.GetClientSets().GetClient()
+
+	// Searching for OOM events that happened before the measurement start.
+	if err := m.handlePriorOOMs(ctx, client); err != nil {
+		return fmt.Errorf("problem with handling prior OOMs: %w", err)
+	}
```

Comment:
Technically you may miss some events between this list and starting informer.

I know it's not that big deal, but you can probably relatively easily reduce the probability by changing the order:
- start the informer
- then call handlePriorOOMs

You can then count something twice potentially, but that's probably not big deal too.

I don't have strong opinion which one is better - but if you want to leave as it, please add a comment that we know that some events might be missed in the meantime and we're fine with that.

Reply:
Good catch. I went with your suggestion of changing the sequence of the function calls.

>You can then count something twice potentially, but that's probably not big deal too.

We already dedupe OOM events by checking the value of the object's RV in the `handleOOMEvent` method.